packages/http-client/src/__tests__/ceramic-http-client.test.ts

```diff
@@ -15,11 +30,81 @@ const initial = {
   ],
 } as unknown as StreamState
 
+const GET_RESPONSE = {
+  ...initial,
+  state: initial,
+}
+
+let did: DID
+let getDidFn
+let client: CeramicClient
```

Comment:
any reason this needs to be a global?  I think you can probably define it locally-scoped within every test